README.md

```diff
@@ -30,6 +30,12 @@ export MANAGEIQ_HOST=localhost
 export MANAGEIQ_PORT=3000
 ```
 
+If the topology inventory requires authentication (ie in dev), basic authentication is supported via these variables. They won't be read in unless :
+```
+export DEV_TOPOLOGICAL_INVENTORY_USERNAME=myuser
```

Comment:
@lindgrenj6 
can we change this to RHN_DEV_USERNAME and RHN_DEV_PASSWORD
Because we need to use the same userid/password in multiple services this is not specific to Topology we would need it for Approval, RBAC and Topology